blocks/subscriptions-block/themes/news.json

```diff
@@ -0,0 +1,301 @@
+{
```

Comment:
These styles should be in `commerce.json` as well, so sites using that theme will have styling on the subs blocks.